### PR TITLE
Bug 2047732: [IBM]Volume is not deleted after destroy cluster

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

**Comment by dobsonj** [comment]
/hold
from the driver logs, [ibm-vpc-block-csi-controller fails to start](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_ibm-vpc-block-csi-driver-operator/40/pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-e2e-ibmcloud-csi/1534116666319835136/artifacts/e2e-ibmcloud-csi/gather-extra/artifacts/pods/openshift-cluster-csi-drivers_ibm-vpc-block-csi-controller-59784c7fd8-dh6qf_iks-vpc-block-driver.log) with this change:
```
flag provided but not defined: -extra-labels
```
@sameshai was the -extra-labels argument added to a newer release of the driver? If so, can you point me to that change so we can get that pulled into the downstream fork before this PR merges?

**Comment by sameshai** [comment]
@dobsonj give me some time these changes are still under testing when everything is finalized, we will provide the details.

**Review by GunaKKIBM — APPROVED** [review]

File: assets/rbac/resizer_role.yaml
Comment (GunaKKIBM) [inline assets/rbac/resizer_role.yaml:24]: adding the resources and permissions for provisioner role should suffice. It can be removed from resizer and registrar.

Comment (GunaKKIBM) [reply assets/rbac/resizer_role.yaml:24]: Please test once, by removing from resizer and regitrar roles.

Comment (GunaKKIBM) [reply assets/rbac/resizer_role.yaml:24]: Please remove it only from resizer role.

**Comment by sameshai** [comment]
> /hold from the driver logs, [ibm-vpc-block-csi-controller fails to start](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_ibm-vpc-block-csi-driver-operator/40/pull-ci-openshift-ibm-vpc-block-csi-driver-operator-master-e2e-ibmcloud-csi/1534116666319835136/artifacts/e2e-ibmcloud-csi/gather-extra/artifacts/pods/openshift-cluster-csi-drivers_ibm-vpc-block-csi-controller-59784c7fd8-dh6qf_iks-vpc-block-driver.log) with this change:
> 
> ```
> flag provided but not defined: -extra-labels
> ```
> 
> @sameshai was the -extra-labels argument added to a newer release of the driver? If so, can you point me to that change so we can get that pulled into the downstream fork before this PR merges?

@dobsonj CSI driver is released v4.4.4. Can you pulled into the downstream fork? Followed this we can merge the operator PR
https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/releases/tag/v4.4.4

**Comment by sameshai** [comment]
@dobsonj can we try the rebase now with 4.4.4

**Comment by dobsonj** [comment]
> I'm running a build now to test all 3 PR's together ([operator](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/40), [driver](https://github.com/openshift/ibm-vpc-block-csi-driver/pull/14), [labeler](https://github.com/openshift/ibm-vpc-node-label-updater/pull/11)), we'll see how that goes.

The cluster came up with storage working, after building an image with all 3 PR's linked above.

**Comment by sameshai** [comment]
> > I'm running a build now to test all 3 PR's together ([operator](https://github.com/openshift/ibm-vpc-block-csi-driver-operator/pull/40), [driver](https://github.com/openshift/ibm-vpc-block-csi-driver/pull/14), [labeler](https://github.com/openshift/ibm-vpc-node-label-updater/pull/11)), we'll see how that goes.
> 
> The cluster came up with storage working, after building an image with all 3 PR's linked above.

Thats great @dobsonj thanks for verifying

**Comment by dobsonj** [comment]
/lgtm
leaving the hold until https://github.com/openshift/ibm-vpc-block-csi-driver/pull/14 and https://github.com/openshift/ibm-vpc-node-label-updater/pull/11 are merged

**Pull request merged**
